### ISSUE TITLE
OCP 4.13 Upgrade Openshift in KLAB

### ISSUE THREAD

**Issue by StevenBarre** [StevenBarre/zh-test-auto#74]
**Describe the issue**
Run the OCP 4.13 upgrade in KLAB. Document if/where it gets stuck and any steps to work around it. Try and determine how disruptive the changes are to the API, Router, and hosted apps.
(Blocked by: Hard Prune in KLAB)

**Additional context**
For all of the involved playbooks, checkout and use the `upgrade` branch to run them from. These branches will be merged into main if no further changes are required.
CCM changes were merged, KLAB needs only to have its PR merged for OCP 4.13 changes to be applied there.

**Definition of done**
- [ ] KLAB upgraded to OCP 4.13
- [ ] MachineConfig changes applied
- [ ] CCM applied
- [ ] EnCase updated
- [ ] ETCD Backup updated
- [ ] Nagios updated
- [ ] Pruner updated
- [ ] vROps updated
- [ ] Logging updated
- [ ] Sample Operator updated
